git_theta/file_io.py

```diff
@@ -2,11 +2,24 @@
 import io
 import json
 import logging
+import zipfile
+import tarfile
 
 from file_or_name import file_or_name
 
 
-def load_tracked_file_from_memory(files):
+def untar_tracked_file_in_memory(file_bytes):
```

Comment:
What in the goal of taring and untaring? I think different goals might lead to different choices of compression libraries

* To make a single object git-lfs stores
* To reduce the disk usage
* To reduce I/O bottlenecks (less data over pipes/to files)

Reply:
Mostly 1 and 3. Conceptually feels a bit simpler if one parameter group corresponds to one lfs object. Also fewer calls to the lfs smudge filter. The disk usage isn't affected since tar is being used without compression.